src/components/EpicView/EpicCard.tsx

```diff
@@ -0,0 +1,199 @@
+import {Issue} from "types";
+import {
+    Avatar,
+    Badge,
+    Box,
+    Center,
+    Grid,
+    Group,
+    Modal,
+    Stack,
+    Text, ThemeIcon,
+    Tooltip,
+    useMantineTheme
+} from "@mantine/core";
+import {Draggable} from "react-beautiful-dnd";
+import {useHover, useMergedRef} from "@mantine/hooks";
+import {DeleteButton} from "../BacklogView/Issue/DeleteButton";
+import {useState} from "react";
+import {useQueryClient} from "@tanstack/react-query";
+import {IconBolt} from "@tabler/icons";
+
+export function EpicCard ({
+    issueKey,
+    summary,
+    status,
+    type,
+    storyPointsEstimate,
+    epic,
+    labels,
+    assignee,
+    index,
+    projectId,
+    ...props
+}: Issue & {index : number}) {
+    let storyPointsColor: string
+    const [opened, setOpened] = useState(false)
+    const queryClient = useQueryClient()
+    const { ref, hovered } = useHover()
+    const theme = useMantineTheme()
+    const hoverStyles =
+        theme.colorScheme === "dark"
+            ? {
+                backgroundColor: theme.colors.dark[8],
+                transition: "background-color .1s ease-in",
+            }
+            : {
+                backgroundColor: theme.colors.gray[1],
+                transition: "background-color .1s ease-in",
+            }
+    switch (status) {
+        case "To Do":
+            storyPointsColor = "gray.6"
+            break
+        case "In Progress":
+            storyPointsColor = "blue.8"
+            break
+        case "Done":
+            storyPointsColor = "green.9"
+            break
+        default:
+            storyPointsColor = "gray.6"
+    }
```

Comment:
Note: Story points box could probably use a refactoring into its own component, but we can add this later.